DANS-KNAW/easy-deposit-api
# EASY-1735 + EASY-1736 DDM convert json to XML including subject/coverages

Fixes EASY-1735

#### When applied it will
* convert most json fields to dataset.xml at submission, ~~still to do:~~
  * [x] subject en coverages  https://drivenbydata.atlassian.net/browse/EASY-1736
  * [x] author IDs
* coordinates for spatial boxes/point changed from number to strings with number format
* [x] exhaustive tests for edge cases
* [x] informative error messages on input errors

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

Submit a dataset with all kinds and variations of content for the metadata, see also `src/test/resources/manual-test`

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
